[Excel] (Custom Functions) Clarify runtimes in UI-less debugging article (#2612)
OfficeDev/office-js-docs-pr: docs/excel/custom-functions-debugging.md

---
ms.date: 04/08/2021
description: Learn how to debug your Excel custom functions that don't use a task pane.
title: UI-less custom functions debugging
localization_priority: Normal
---
# UI-less custom functions debugging

> Debugging for custom functions that don't use a task pane or other user interface elements (UI-less custom functions) can be accomplished by multiple means, depending on what platform you're using. 

Reviewer: ```suggestion
This article discusses debugging *only* for custom functions that don't use a task pane or other user interface elements (UI-less custom functions). 
```

## Requirements

> This debugging process only works for UI-less custom functions, which are custom functions that use the `OfficeRuntime` JavaScript runtime. This debugging process doesn't work with custom functions that use the `SharedRuntime` JavaScript runtime, or with custom functions that use the `CustomFunctionsRuntime` JavaScript runtime. 

Reviewer: ```suggestion
This debugging process only works for UI-less custom functions, which are custom functions that use the `OfficeRuntime` JavaScript runtime. This debugging process doesn't work with custom functions that use the `SharedRuntime` JavaScript runtime. 
```
